components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/ActionExecutionLogConstants.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.action.execution;
+
+/**
+ * Diagnostics Logs Constants for Action execution service.
+ */
+public class ActionExecutionLogConstants {
+
+    private ActionExecutionLogConstants() {}
+
+    public static final String ACTION_EXECUTION = "action-execution";
```

Comment:
shall we make this constant name as 'ACTION_EXECUTION_COMPONENT_ID'. Because that's what this means in the diagnostic log context. And given this class is exposed for other components to use the name should be descriptive enough to understand where it should be used

Reply:
Updated in https://github.com/wso2/carbon-identity-framework/pull/5969/commits/18cd22891c51dc80a1674aea605ea1617165a529